lua/neogit/lib/git/branch.lua

```diff
@@ -1,4 +1,5 @@
 local a = require 'plenary.async_lib'
+--luacheck: ignore void
```

Comment:
why is this needed?

Reply:
Because "void" is unused. It can also just be removed

Reply:
Unused imports can be safely removed imo.